src/sentry/static/sentry/app/styles/card.tsx

```diff
@@ -0,0 +1,47 @@
+import {css} from '@emotion/core';
+
+import {Theme} from 'app/utils/theme';
+
+type CardStyleProps = {
+  theme: Theme;
```

Comment:
Please, why not just import the theme as we are doing?

Reply:
This is the Theme type no?

Reply:
you have a theme as a prop 😉

Reply:
Oh, so we can be proper about theme passing, in case we ever do have a `dark` theme.